Dockerfile

```diff
@@ -20,7 +20,7 @@ git \
 subversion \
 maven \
 && microdnf -y clean all
-ARG WINDUP=https://repo1.maven.org/maven2/org/jboss/windup/tackle-cli/6.1.2.Final/tackle-cli-6.1.2.Final-offline.zip
+ARG WINDUP=https://oss.sonatype.org/content/groups/public/org/jboss/windup/windup-cli/6.2.0-SNAPSHOT/windup-cli-6.2.0-20230302.112956-104-offline.zip
```

Comment:
This can merge as-is but will need a follow-up PR when the FINAL (official) release is available.
This MUST be done before the next tackle release.